js/ai/src/prompt.ts

```diff
@@ -736,6 +736,12 @@ function loadPrompt(
       if (variant) {
         promptMetadata.variant = variant;
       }
+
+      // dotprompt can set null description on the schema, which can confuse downstream schema consumers
+      if (promptMetadata.output?.schema?.description === null) {
+        delete promptMetadata.output?.schema?.description;
+      }
```

Comment:
I haven't looked that close, but why does description in particular have this problem? Do other fields also have this problem?